cmd/buck/cli/push.go

```diff
@@ -20,6 +20,11 @@ import (
 	"github.com/textileio/textile/cmd"
 )
 
+var (
+	buckMaxSizeMiB = int64(1024)
```

Comment:
So this is 1Gib. Which is the same as the default total max size of buckets in an account.
That's to say, in the CLI we're covering to worst of the worst case: a single bucket eating the whole *total* size quota of the account.